cli/test/action/action.go

```diff
@@ -204,8 +201,21 @@ func executeUntil(ctx context.Context, args []string, until string) (string, err
 		if err != nil {
 			panic(err)
 		}
+		isDoneWg.Done()
 	}()
 
+	originalWait := s.Wait
+	s.Wait = func() {
+		originalWait()
+		// Because we are overwriting a singleton (os.Stderr) we must wait for the full command
+		// to finish executing before we allow the next test to execute, else the next test may
+		// try and overwrite os.Stderr before this execution finishes closing down the database.
+		//
+		// That might not cause any real bother, but the go test `-race` flag will complain.
+		isDoneWg.Wait()
+		os.Stderr = stdErr
```

Comment:
question: Why is this needed now but not before?

Reply:
Because it can be, before we were essentially piping pipes - this way we actually reset the state (this line is not required for the fix)